recipes/osmanip/all/conanfile.py

```diff
@@ -81,6 +80,9 @@ def validate(self):
             # test_package segfaults with libstdc++ for some reason
             raise ConanInvalidConfiguration("osmanip >= 4.5.0 doesn't support libstdc++")
 
+        if is_msvc(self):
+            raise ConanInvalidConfiguration("MSVC is not yet supported by osmanip recipe. Contributions are welcome.")
```

Comment:
this is not a regression. msvc was not built by previous recipes because of [arsenalgear](https://github.com/conan-io/conan-center-index/blob/master/recipes/arsenalgear/all/conanfile.py#L71)